mrp_sale_info/models/mrp_production.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2016 Antiun Ingenieria S.L. - Javier Iniesta
+# Copyright 2019 Rubén Bravo <rubenred18@gmail.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class MrpProduction(models.Model):
+    _inherit = "mrp.production"
+
+    sale_id = fields.Many2one(
```

Comment:
`sale_id` is a duplication of `sale_order_id` [here](https://github.com/OCA/sale-workflow/blob/12.0/sale_mrp_link/models/mrp_production.py#L11) in module `sale_mrp_link`.
I think it's better to avoid duplication and preserve only one of these, the latter as is defined in a module in use.

Reply:
@sergiocorato the link in the other module is very weak (through a search of a string in origin). This one has been here for a long and IMO this is the definitive one to be preserved. Also the typical name for this field in other places is `sale_id`, and not `sale_order_id`.

Reply:
@pedrobaeza Ok, perhaps coordinate https://github.com/OCA/sale-workflow/pull/1109?